lib/categories/Math/MathRiddlelLists.dart

```diff
@@ -23,135 +19,149 @@ class MathRiddles extends StatefulWidget {
-                  onPressed: () {
-                    Navigator.of(context).push(
-                        MaterialPageRoute(builder: (ctx) => Math4HomeScreen()));
-                  }),
-              ActionCard(
-                  name: 'OBSERVE & COUNT',
-                  description:
-                      'A MATH RIDDLE TO WHICH OBSERVATION GIVES THE COUNT\nby @shobhit907',
-                  number: '5',
-                  onPressed: () {
-                    Navigator.of(context).push(
-                        MaterialPageRoute(builder: (ctx) => Math5HomeScreen()));
-                  }),
-              ActionCard(
-                  name: 'GREATEST BINARY',
-                  description:
-                      'A MATH RIDDLE TO FIND GREATEST BINARY NUMBER\nby @Denish-Ranpariya',
-                  number: '6',
-                  onPressed: () {
-                    Navigator.of(context).push(
-                        MaterialPageRoute(builder: (ctx) => Math6HomeScreen()));
-                  }),
-              ActionCard(
-                  name: 'NUMBER OF TICKETS',
-                  description: 'A MATH BRAIN TEASER\nby @MoyinShabi',
-                  number: '7',
-                  onPressed: () {
-                    Navigator.of(context).push(
-                        MaterialPageRoute(builder: (ctx) => Math7HomeScreen()));
-                  }),
-              ActionCard(
-                  name: 'ODD <-> EVEN',
-                  description: 'Odd Even conversion\nby @c0decreat0r',
-                  number: '8',
-                  onPressed: () {
-                    Navigator.of(context).push(
-                        MaterialPageRoute(builder: (ctx) => Math8HomeScreen()));
-                  }),
-              ActionCard(
-                  name: 'WRONG BUT RIGHT',
-                  description:
-                      'THINK A LITTLE OUT OF THE BOX FOR THIS ONE\nby @razorcalhn',
-                  number: '9',
-                  onPressed: () {
-                    Navigator.of(context).push(
-                        MaterialPageRoute(builder: (ctx) => Math9HomeScreen()));
-                  }),
-              ActionCard(
-                  name: 'Magic Number',
-                  description:
-                      'A math riddle which is very fun to try out by @bibekpanda55',
-                  number: '10',
-                  onPressed: () {
-                    Navigator.of(context).push(MaterialPageRoute(
-                        builder: (ctx) => Math10HomeScreen()));
-                  }),
-              ActionCard(
-                  name: 'Card Skills',
-                  description:
-                      'A math riddle based on the card deck by @mayankparida',
-                  number: '11',
-                  onPressed: () {
-                    Navigator.of(context).push(MaterialPageRoute(
-                        builder: (ctx) => Math11HomeScreen()));
-                  }),
-
-              ActionCard(
```

Comment:
12 and 13 number Riddles will be removed if I merge this PR, So can you Pull these files and then send the PR again?😶

Reply:
Added those back in, thanks for catching those!